LiskHQ/lisk-hub
# Dont allow to call API when going on Delegate statistics page - Closes #879

### What was the problem?
Clicking on `Delegate Statistic` tab was triggering unnecessary API call with the wrong filter value
### How did I fix it?
Changed action for additional tabs
### How to test it?
Open `Network` console and change tab t o`Delegate Statistics` check if it triggers any API call
### Review checklist
- The PR solves #879
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
